tsconfig.json

```diff
@@ -15,10 +15,29 @@
     "strict": true,
     "target": "es5",
     "noImplicitAny": false,
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
     "allowJs": true,
-    "isolatedModules": true
+    "isolatedModules": true,
+    "paths": {
+      "@emotion/core": [
+        "./types/index.d.ts"
+      ],
+      "@storybook/theming": [
+        "./types/index.d.ts"
+      ]
```

Comment:
@paulocfjunior Yes they do.

Because `storybook-theming` relies on some `@emotion` packages, we have this dependency.

Storybook enforces global types for his theming package as mentioned [here](https://github.com/modulz/stitches/issues/897#issuecomment-996474061):
> AFAIK Storybook has defined global type definitions for Emotion's css prop which clash with Stitches' css prop. A work around is to point the Emotion package at an empty types definition file.